pages/contact-us.tsx

```diff
@@ -1,7 +1,63 @@
 import { NextPage } from "next";
+import { SocialIcon } from "react-social-icons";
 
 const ContactUsPage: NextPage = () => {
-  return <div>Contact us page</div>;
+  return <div className='container max-w-xl mx-auto py-12 md:max-w-4xl'>
+    <div>
+      <h1 className='text-4xl font-medium'>Please get in touch and our expert support team will answer all your
+        questions.
+      </h1>
+    </div>
+    <div className='content flex md:flex-row flex-col gap-x-6 mt-12'>
+      <div className='max-w-xl'>
+        <div className="flex flex-col gap-8">
+          <label htmlFor="" className="block">
+            <span className="text-gray-700">Full name</span>
+            <input type="text" className="mt-1 block w-full" placeholder=""/>
+          </label>
+          <label htmlFor="" className="block">
+            <span className="text-gray-700">Email</span>
+            <input type="email" className="mt-1 block w-full" placeholder="john@example.com"/>
+          </label>
+          <label htmlFor="" className="block">
+            <span className="text-gray-700">Your message</span>
+            <textarea className="mt-1 block w-full"></textarea>
+          </label>
+          <button className="thd-button-secondary bg-[#1da1f2] text-white text-xl font-medium p-2 rounded">Send</button>
```

Comment:
Where does the `thd-button-secondary` class name come from?

Reply:
This class is a product of my immature mind. I'll remove it with the next pr.

Reply:
🤔 But I don't see it defined anywhere? Does it have any effect?

Reply:
No.

Reply:
So remove it? 🤔

Reply:
It's redundant then, isn't it?